app/app.py

```diff
@@ -287,6 +286,7 @@ def api_events():
                     "activity": event.activity,
                     "start_time": event.start_time,
                     "period": event.period,
+                    "can_complete": str(can_complete(event, datetime.datetime.now())),
```

Comment:
Avoid calling `datetime.now()` multiple times. Put it at the beginning of the if-statement stored in variable and use it here.

Also, don't convert `bool` to `str`, remove outer `str()` call.